### PR TITLE
Qa/update circle config

### PR DESCRIPTION
**Pull request by kspeer825**
# Description of change
Get the build passing, mark pylint issues.
# QA steps
 - [x] automated tests passing
 - [x] manual qa steps passing (list below)
  - remove the variable from the circleci-user context
  - ensure that the pylint disables are logged in the run output
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,8 +18,11 @@ jobs:
       - run:
           name: 'pylint'
           command: |
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox dev_env.sh
```
Comment (luandy64) [inline .circleci/config.yml:21]: Should we move this into some directory that we can persist for any other step to reuse?

File: .circleci/config.yml [hunk 2]
```diff
@@ -18,8 +18,11 @@ jobs:
...
+            source dev_env.sh
             source /usr/local/share/virtualenvs/tap-google-analytics/bin/activate
-            pylint tap_google_analytics --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+            # BUG | https://jira.talendforge.org/browse/TDL-15395
+            pylint tap_google_analytics --disable '{$PYLINT_DISABLE_LIST},line-too-long, fixme, missing-module-docstring, missing-function-docstring, too-many-locals, invalid-name, broad-except, missing-class-docstring, too-many-arguments, no-else-return'
```
Comment (luandy64) [inline .circleci/config.yml:25]: Interesting. I thought this would have needed double quotes

**Review by luandy64 — COMMENTED** [review]
Looks good, I just have a quick question

File: .circleci/config.yml (hunk 1 above)
Comment (kspeer825) [reply .circleci/config.yml:21]: Yeah we should, I was being lazy

File: .circleci/config.yml (hunk 2 above)
Comment (kspeer825) [reply .circleci/config.yml:25]: It was single quotes before

File: .circleci/config.yml (hunk 1 above)
Comment (luandy64) [reply .circleci/config.yml]: It just becomes a question of whether we think the added complexity is worth it for a file we source twice

File: .circleci/config.yml (hunk 2 above)
Comment (luandy64) [reply .circleci/config.yml]: I just don't understand bash 🤷🏻‍♂️

File: .circleci/config.yml (hunk 1 above)
Comment (kspeer825) [reply .circleci/config.yml]: So for single job workflows, there's no need to generate a tmp workspace. We can access files across runs. I just put the cp in the env step of the job.

File: .circleci/config.yml (hunk 2 above)
Comment (kspeer825) [reply .circleci/config.yml]: You were correct. Double quotes are needed else we won't be using the variable properly in the pylint call.

**Review by luandy64 — APPROVED** [review]

**Pull request merged**